mpesa/LipaNaMpesaOnline.py

```diff
@@ -5,45 +5,20 @@
 from requests.auth import HTTPBasicAuth
 from base64 import b64encode
 from .models import PaymentTransaction
-from cryptography import x509
-from cryptography.hazmat.backends import default_backend
-from cryptography.hazmat.primitives.asymmetric.padding import PKCS1v15
 
 # import env settings
-import importlib
-
-try:
-    dotenv = importlib.import_module('dotenv')
-    dotenv.load_dotenv(dotenv.find_dotenv())
-except ModuleNotFoundError:
-    pass
-
-consumer_key = os.environ.get('CONSUMER_KEY')
-consumer_secret = os.environ.get('CONSUMER_SECRET')
-api_URL = os.environ.get('API_URL')
-INITIATOR_PASS = os.environ.get('INITIATOR_PASS')
-CERTIFICATE_FILE = os.environ.get('CERTIFICATE_FILE')
-HOST_NAME = os.environ.get('HOST_NAME')
-PASS_KEY = os.environ.get('PASS_KEY')
-shortcode = os.environ.get('SHORT_CODE')
-SAFARICOM_API = os.environ.get('SAFARICOM_API')
-
-
-def encryptInitiatorPassword():
-    PASS = "foobar1234"
-    cert_file_path = os.path.join(os.path.dirname(__file__), CERTIFICATE_FILE)
-    cert_file = open(cert_file_path, 'rb')
-    cert_data = cert_file.read()
-    cert_file.close()
-
-    cert = x509.load_pem_x509_certificate(cert_data, default_backend())
-    PASS = str.encode(PASS)  # If you are using python 3
-    pub_key = cert.public_key()
-    cipher = pub_key.encrypt(PASS, padding=PKCS1v15())
-    return b64encode(cipher)
-
-
-def generate_pass_key(cipher):
+from decouple import config
```

Comment:
python-decouple has not been added to the requirements

Reply:
Coz that's where the whole LipaNaMpesa Online payment method functionality lies. I forgot to do a pip freeze -r requirements.txt But I guess Paul can ensure that it's in the requirements.txt file.

Reply:
@Ekirapapaul what do you think of the name change. Will there be anotther file for BuyGoods etc..

Reply:
@muneneevans  I am okay with the name change. We will support other services on separate files. I think it is also okay to decouple the functionality.

@Ronnie I will add the requirement but just to point out in future it is better for the author to include everything. Otherwise good work on the PR. Thanks.